docs/INSTALL-devel.rst

```diff
@@ -297,7 +297,7 @@ Start the server
 
 Run the server with the port number as defined in ``hil.cfg``::
 
-  hil-admin run-dev-server <port no>
+  hil-admin run-dev-server
```

Comment:
Good catch. The port number can be specified with the `--port` flag. Without it always defaults to 5000. Maybe change it to `hil-admin run-dev-server --port <port no>`.
```
(.venv) naved::~/hil-all/hil$hil-admin run-dev-server --help
usage: hil-admin run-dev-server [-?] [--port PORT]

Run a development api server. Don't use this in production. Specify the port
with -p or --port otherwise defaults to 5000

optional arguments:
  -?, --help            show this help message and exit
  --port PORT, -p PORT
(.venv) naved::~/hil-all/hil$hil-admin run-dev-server -p 6000
INFO:werkzeug: * Running on http://127.0.0.1:6000/
^C(.venv) naved::~/hil-all/hil$hil-admin run-dev-server
INFO:werkzeug: * Running on http://127.0.0.1:5000/
```

Reply:
I second this